templates/springboot/http/src/main/java/functions/CloudFunctionApplication.java

```diff
@@ -0,0 +1,38 @@
+package functions;
+
+import java.util.function.Function;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class CloudFunctionApplication {
+
+  public static void main(String[] args) {
+    SpringApplication.run(CloudFunctionApplication.class, args);
+  }
+
+  @Bean
+  public Function<String, String> uppercase() {
+    return String::toUpperCase;
+  }
+
+  /**
+   * Health checks
+   * 
+   * @return
+   */
+  @Bean
+  public Function<String, String> health() {
```

Comment:
The SB/Spring Cloud Function does not add *probes* by default, usually in SB world we add it via spring-actuator which adds healthchecks. Since we use Spring Cloud Function we need to add them as above. I discovered that during my tests today when the Knative Service was Crashing because missing probes `/health/liveness` and `/health/readiness`.